ee_tests/README.md

```diff
@@ -66,7 +81,7 @@ NODE_DEBUG=true npm start
 DEBUG=true npm start
 ```
 
-By default, Google Chrome browser is used by EE tests via Selenium. If you want
+By default, Google Chrome browser is used by E2E tests via Selenium. If you want
```

Comment:
I am not sure we should be mentioning Firefox since no one has tried that in a very long time and it is not guaranteed to work. Also, we do not actually use Selenium any more, do we?

Reply:
Protractor is using Selenium internally, I didn't want to touch other sections, just unify our acronyms.